api/scripts/create-target-profile-trainings.js

```diff
@@ -0,0 +1,66 @@
+const { knex, disconnect } = require('../db/knex-database-connection');
+const trainingRepository = require('../lib/infrastructure/repositories/training-repository');
+const targetProfileRepository = require('../lib/infrastructure/repositories/target-profile-repository');
+const { NotFoundError } = require('../lib/domain/errors');
+
+async function checkTrainingExistence(trainingId) {
+  try {
+    await trainingRepository.get(trainingId);
+  } catch (error) {
+    throw new NotFoundError(`Training ${trainingId} not found`);
+  }
+}
+
+async function checkTargetProfileExistence(targetProfileId) {
+  try {
+    await targetProfileRepository.get(targetProfileId);
+  } catch (error) {
+    throw new NotFoundError(`Target profile ${targetProfileId} not found`);
+  }
+}
+
+async function createTargetProfileTraining(targetProfileTraining) {
+  return knex.batchInsert('target-profile-trainings', targetProfileTraining);
```

Comment:
```suggestion
  return knex.batchInsert('target-profile-trainings', targetProfileTraining);
```
Question: Pourquoi un batch pour un seul training ?